node_modules/col-assets/tests/test-migrate.js

```diff
@@ -130,11 +133,44 @@ describe('Migrate', function() {
     /**
      * Test that verifies migration from legacy filesystem to Amazon S3
      */
-    it('migrates file from Amazon S3 to Amazon S3', function(callback) {
+    it('migrates files from Amazon S3 to Amazon S3', function(callback) {
+      var course = TestsUtil.generateCourse(global.tests.canvas.ucberkeley);
+      var instructor = TestsUtil.generateInstructor(global.tests.canvas.ucberkeley);
+      var categoryName = 'Category 1';
+
+      // Instructor is teaching course with files in Canvas filesystem
+      AssetsTestUtil.setUpAmazonS3BackedCourse(instructor, function(client1, course1, instructor1) {
+        CategoriesTestUtil.assertCreateCategory(client1, course1, categoryName, function(category) {
+
+          // Put file to Canvas filesystem
```

Comment:
@johncrossman This comment and the one above imply that course1 is storing its files in Canvas, but is that actually true?